components/datepicker/stories/datepicker.mdx

```diff
@@ -0,0 +1,74 @@
+import { Canvas, ArgTypes, Meta, Description, Title } from '@storybook/blocks';
+
+import * as DatePickerStories from './datepicker.stories';
+
+<Meta of={DatePickerStories} title="Docs" />
+
+<Title of={DatePickerStories} />
+<Description of={DatePickerStories} />
+
+## Usage notes
+### General notes
+- Date picker uses `.spectrum-PickerButton` instead of a `.spectrum-Picker`.
+- Workflow icon size is medium. If your markup is still using the small icon, replace `.spectrum-Icon--sizeS` with `.spectrum-Icon--sizeM`.
+
+### Component separated from InputGroup
+This component was previously a variant within InputGroup. **InputGroup is deprecated**.
+
+These classes containing `InputGroup` have been renamed
+
+- v1.0.0: All date picker classes have been renamed to have a capital P: `.spectrum-Datepicker` is now `.spectrum-DatePicker`
+- v1.0.0: `.spectrum-InputGroup-textfield` is now `.spectrum-DatePicker-textfield`, `.spectrum-InputGroup-input` is now `.spectrum-DatePicker-input`, and `.spectrum-InputGroup-button` is now `.spectrum-DatePicker-button`
+- `.spectrum-Datepicker--rangeDash` was renamed to `.spectrum-DatePicker-rangeDash`
+
+And these have been removed:
+- v1.0.0: The class `.InputGroup` is no longer used and can be be removed from the parent element
+- `.spectrum-Datepicker-focusRing` is no longer required and should be removed
```

Comment:
Is it worth it to try to re-format this section to better mirror the same section in combobox (with the `->`)?  It just seems more straightforward to me.

BUT- that's not how it is on the docs site, and this one has a some extra info about the version numbers, so it might not be possible or necessary to do in this PR.

Reply:
Good point! I'll reformat 👍